lib/import-export-visitor.js

```diff
@@ -468,6 +525,8 @@ module.exports = class ImportExportVisitor extends Visitor {
 
         path.replace(this._buildExportDefaultStatement(dd));
       }
+
+      this.madeChanges = true;
```

Comment:
Here's one case where we set `this.madeChanges = true` even though nothing is being hoisted. Specifically, this means we still add `"use strict"` when you have only `export default someExpression` and no other kinds of `import` or `export` declarations.